src/common/dtos/TotalsDTO.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+export interface TotalDataDTO {
+  revenue: { paid_to_date: string; code: string };
+  expenses: { amount: string; code: string };
+  outstanding: Record<string, unknown>;
+}
+export interface ChartDataDTO {
+  invoices: { total: string; date: string; currency: string }[];
+  payments: { total: string; date: string; currency: string }[];
+  expenses: {
+    total: string;
+    date: string;
+    currency: string;
+  }[];
+}
+export interface ChartMapDTO {
+  name: string;
+  invoices: string;
+  expenses: string;
+  payments: string;
+}
+export interface RequestBodyDTO {
+  start_date: string;
+  end_date: string;
+}
```

Comment:
I don't think there's generally a need for this. It's good to have this stuff typed, but definitely no need to be in common/ because I don't think we'll be using this same interface anywhere else.

Also `DTO` suffix is an anti-pattern.